.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,8 +3,8 @@ quickly and easily:
 
  - [ ] Make sure the PR title is formatted like:
    `[BEAM-<Jira issue #>] Description of pull request`
- - [ ] Make sure tests pass via `mvn clean verify`. (Even better, enable
-       Travis-CI on your fork and ensure the whole test matrix passes).
+ - [ ] Make sure tests pass via `mvn clean install -Pall-checks`. (Even better,
```

Comment:
On the other hand, to argue against myself here a bit, if it fails in Jenkins it's way easier in the case of a new contributor for them to get help from their reviewer. No pastebinning or anything of console output from their computer, publicly-accessible by default.